frontend/webservice/vrf.cgi

```diff
@@ -842,6 +894,18 @@ sub provision_vrf{
         $reason = "Created by $ENV{'REMOTE_USER'}";
     }
 
+    my $error = OESS::DB::VRF::add_vrf_history(
+        db => $db,
+        event => $type eq 'provisioned' ? 'create' : 'edit',
+        vrf => $vrf,
+        user_id => $user->user_id,
+        workgroup_id => $params->{workgroup_id}{value},
+        state => 'decom'
+    );
+    if (defined $error) {
+        warn $error;
+    }
+
```

Comment:
This should be called prior to `$db->commit` to be run within the same transaction